packages/ethereum-contracts/contracts/upgradability/BeaconProxiable.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: AGPLv3
+pragma solidity 0.8.19;
+
+import { Initializable } from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+
+abstract contract BeaconProxiable is Initializable {
+
+    // allows to mark logic contracts as initialized in order to reduce the attack surface
```

Comment:
> reduce the attack surface

->

-> +, also to simplify the life for bug bounty white hacker by asking them to look else where. Because we understand that there should be little to no attack surface without initializer.

Reply:
Same to the UUPSProxiable

Reply:
addressed